src/spatialCapture/Shaders.js

```diff
@@ -314,7 +320,8 @@ export function createPointCloudMaterial(texture, textureDepth, shaderMode, bord
             viewAngleSimilarity: { value: 0.0 },
             viewPositionSimilarity: { value: 0.0 },
 			// used to erase depth in the pre render
-            useFarDepth: {value: false}
+            useFarDepth: {value: false},
+            inverted: {value: false}
```

Comment:
Do we not want it always set to true?

Reply:
The live video is not inverted, keeps it interesting :D